door_button.py

```diff
@@ -124,7 +124,8 @@ def main():
                 person['name'],
                 availibility,
                 (person.get('pos_x', 0), person.get('pos_y', 0)),
-                person.get('offset', 70), "assets/{}".format(person.get('image', 'unknown.png'))
+                (person.get('ballon_x', 70), person.get('ballon_y', -50)),
```

Comment:
Default value set to -70 is not good if they are supposed to be absolute coordinates